.github/workflows/on_pull_request.yaml

```diff
@@ -15,15 +15,15 @@ jobs:
     name: test
     strategy:
       matrix:
-        go-version: [ 1.18.x, 1.19.x ]
+        go-version: [ oldstable, stable ]
```

Comment:
Apologies for the delay. VulcanD isn't supported much these days and I didn't get the ping.

I'm not sure this is actually helpful for a project like VulcanD given how little activity it gets. The versions will likely skip many times before a new PR comes along to actually test new versions. If it broke somewhere in-between, how will you know?

Reply:
Fair point. Let's close it then 🤗